### ISSUE TITLE
Interactive mode doubles entity

### ISSUE THREAD

**Issue by slesinger** [RasaHQ/rasa#7533]
**Rasa version**:
2.1.2
**Rasa SDK version** (if used & relevant):
2.1.2
**Rasa X version** (if used & relevant):
0.33.2
**Python version**:
3.8.5
**Operating system** (windows, osx, ...):
Ubuntu 20.04 Linux-5.4.0-54-generic-x86_64-with-glibc2.29
**Issue**:
During interactive mode, Rasa correctly recognizes entity in users input however enity is doubled:
```
Type a message and press enter (press 'Ctr-c' to exit).
? Your input -> zhasni v ložnici                                                                                        
? Is the intent 'zhasnout' correct for 'zhasni v [ložnici]{"entity": "mistnost", "value": "loznice"}[ložnici]{"entity": 
"mistnost", "value": "loznice"}' and are all entities labeled correctly?  (Y/n)          
```
Saved output to nlu.yaml reflects the same
```
    - zhasni v [ložnici]{"entity": "mistnost", "value": "loznice"}[ložnici]{"entity": "mistnost", "value": "loznice"}
```
Expected outcome is
```
    - zhasni v [ložnici]{"entity": "mistnost", "value": "loznice"}
```
**Error (including full traceback)**:
No error

**Command or request that led to error**:
N/A

**Content of configuration file (config.yml)** (if relevant):
Likely irrelevant

**Content of domain file (domain.yml)** (if relevant):
Likely irrelevant

**Comment by sara-tagger** [comment]
Thanks for raising this issue, @dakshvar22 will get back to you about it soon✨

###### Please also check out the [docs](https://rasa.com/docs/) and the [forum](https://forum.rasa.com/) in case your issue was raised there too 🤗

**Comment by samsucik** [comment]
@slesinger could you describe your setup in more detail? I see you use also Rasa X, was that used to train the model or for something else? Additionally, what does your NLU training data for the intent `zhasnout` look like?  We can try to reproduce the error because it looks like it could be a bug...

**Comment by slesinger** [comment]
Hello @samsucik, I have used RasaX initially, but then I switched to edit plain files. To give you full picture I published my bot here: https://github.com/slesinger/assistant.git . NLU data are in czech language. I am sorry, that was original intent. Anyway, my gut feeling is that this issues is not related to language used. Thanks.

**Comment by samsucik** [comment]
I'll take look. By the way, I understand Czech, so that shouldn't be a huge issue 🙂

**Comment by samsucik** [comment]
Alright, @slesinger so the reason for this weird behaviour is that you included in your config file two CRF entity extractors. One as the `CRFEntityExtractor` pipeline component and other instance is `DIETClassifier` (the component which, by default, does both intent classification and entity extraction using CRF).

If you remove `CRFEntityExtractor` from the config, entities get picked only once.

For the future, maybe Rasa should give a warning or not produce duplicated annotations when there are multiple instances of an entity extractor included in the config, what do you think? I encourage you to create a new issue here on GitHub if you'd like to propose some changes/improvements 🙂

**Comment by slesinger** [comment]
Thanks so much. It works as expected after I have removed ```CRFEntityExtractor```. 
I am providing my ```config.yml``` for reference in case somebody has the same issue. At least, it will be documented.

```
language: cs
pipeline:
- name: WhitespaceTokenizer
- name: RegexFeaturizer
- name: CRFEntityExtractor              <--------- to be removed
- name: LexicalSyntacticFeaturizer
- name: CountVectorsFeaturizer
- name: CountVectorsFeaturizer
  analyzer: char_wb
  min_ngram: 1
  max_ngram: 4
- name: DIETClassifier
  epochs: 100
- name: EntitySynonymMapper
- name: ResponseSelector
  epochs: 100
- name: FallbackClassifier
  threshold: 0.7

policies:
- name: MemoizationPolicy
- name: TEDPolicy
  max_history: 5
  epochs: 30
- name: RulePolicy
  core_fallback_threshold: 0.4
  core_fallback_action_name: "action_default_fallback"
  enable_fallback_prediction: True
```
